frontend/packages/pipelines-plugin/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx

```diff
@@ -55,15 +56,15 @@ describe('PipelineRunDetailsPage:', () => {
   it('Should contain task runs page', () => {
     const { pages } = wrapper.props();
 
-    const taskRunsPage = pages.find((page) => page.name === 'TaskRuns');
+    const taskRunsPage = pages.find((page) => page.nameKey === `${i18nPipelineNS}~TaskRuns`);
```

Comment:
Your change is okay! Alternative you check if the page name includes the right value:

```suggestion
    const taskRunsPage = pages.find((page) => page.nameKey.includes('TaskRuns');
```